MSBuild.Sdk.Extras.Tests/MSBuild.Sdk.Extras.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<!--<Import Project="$(MSBuildThisFileDirectory)..\MSBuild.Sdk.Extras\build\MSBuild.Sdk.Extras.props" />-->
+
+	<PropertyGroup>
+		<!--<TargetFramework>uap10.0</TargetFramework>-->
+		<TargetFrameworks>uap10.0;monoandroid70;xamarin.ios10;xamarin.mac20</TargetFrameworks>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="MSBuild.Sdk.Extras" Version="1.2.0" PrivateAssets="All" />
```

Comment:
Is there a way to test as `PackageReference` in CI, rather than a static reference to props and targets?